v2/concordium/service.proto

```diff
@@ -122,4 +122,19 @@ service Queries {
 
   // Get the current branches of blocks starting from and including the last finalized block.
   rpc GetBranches (Empty) returns (Branch);
+
+  // Get information related to the baker election for a particular block.
+  rpc GetElectionInfo (BlockHashInput) returns (ElectionInfo);
+
+  // Get the identity providers registered as of the end of a given block.
+  // The stream will end when all the identity providers have been returned.
+  rpc GetIdentityProviders (BlockHashInput) returns (stream IpInfo);
+
+  // Get the anonymity revokers registered as of the end of a given block.
+  // The stream will end when all the anonymity revokers have been returned.
+  rpc GetAnonymityRevokers (BlockHashInput) returns (stream ArInfo);
+
+  // Get a list of non-finalized transaction hashes for a given account.
+  // The stream will end when all the non-finalized transaction hashes have been returned.
```

Comment:
What is the rationale for streaming here? In practice this will almost always be very few.

I don't necessarily object to this, but I would like to hear why.